### PR TITLE
Showcase cards 2

### PR DESCRIPTION
**Pull request by MaciejLabedzkiCodewave**

### CODE REVIEW

File: src/components/ShowcaseCard/ShowcaseCard.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+ */
+const ShowcaseCard = ({
+    title,
+    exceptr,
+    src,
+    srcAlt,
+    githubUrl,
+    predefinedCTUrl,
+    starterPageUrl,
+    demoUrl,
```
Comment (trzcina) [inline src/components/ShowcaseCard/ShowcaseCard.js:42]: We need to discuss the naming convention. If it is possible, the ShowcaseCard component should be the all-purpose component, not only listing templates on our site.

File: src/stories/ShowcaseCard.stories.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <ShowcaseCard {...args} />
+);
+
+export const Standard = Template.bind({});
+// More on args: https://storybook.js.org/docs/react/writing-stories/args
+Standard.args = {
+    title: 'Kyan - blog screenshot',
+    // eslint-disable-next-line max-len
+    exceptr: 'The total package with which you can build beautiful, high converting and professional The total package witch you can build',
+    src: 'https://api.flotiq.com/image/0x0/_media-3a62e398-fbe3-46e8-ba5b-05a2690390ae.png',
```
[the remark this answers is not in the document]
Comment (MaciejLabedzkiCodewave) [reply src/stories/ShowcaseCard.stories.js:25]: Asset removed.

File: src/components/ShowcaseCard/ShowcaseCard.js [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
++ 'DExLjcyNTkgOC4yNzQxMSAxMy4xMjUgMTAgMTMuMTI1WiIgc3Ryb2tlPSJibGFjayIgc3'
++ 'Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtbGluZWpvaW49InJvdW5kIi8+Cjwvc3Z'
++ 'nPgo=';
+
+/**
+ * Component for displaying product showcase card
+ */
+const ShowcaseCard = ({
+    title,
+    exceptr,
```
[the remark this answers is not in the document]
Comment (MaciejLabedzkiCodewave) [reply src/components/ShowcaseCard/ShowcaseCard.js]: Typo fix from exceptr to exceptr

File: src/components/ShowcaseCard/ShowcaseCard.js [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+    src,
+    srcAlt,
```
[the remark this answers is not in the document]
Comment (MaciejLabedzkiCodewave) [reply src/components/ShowcaseCard/ShowcaseCard.js]: Rename: src to thumbnailSrc and srcAlt to thumbnailSrcAlt

File: src/components/ShowcaseCard/ShowcaseCard.js [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+
+        <div className="w-full aspect-w-1 aspect-h-1
+        bg-gray-200 overflow-hidden xl:aspect-w-7 xl:aspect-h-8 relative"
+        >
+            <a
+                href={starterPageUrl}
+                target="_blank"
+                rel="noreferrer"
+            >
+                <div className="absolute w-full top-0 left-0 h-full z-index-1
```
[the remark this answers is not in the document]
Comment (MaciejLabedzkiCodewave) [reply src/components/ShowcaseCard/ShowcaseCard.js:77]: Transition animation added.

File: src/components/ShowcaseCard/ShowcaseCard.js (hunk 1 above)
Comment (MaciejLabedzkiCodewave) [reply src/components/ShowcaseCard/ShowcaseCard.js]: Remove **githubUrl** and **predefinedCTUrl**. 
Replace them with **footerLinks : [{link:***,text:***}]**.
 
image

Comment (MaciejLabedzkiCodewave) [reply src/components/ShowcaseCard/ShowcaseCard.js]: Replace **starterPageUrl** with **cardUrl**.

image

**Pull request merged**